scripts/sanity_chk/harness.py

```diff
@@ -121,6 +127,29 @@ def handle(self, line):
 
         self.process_test(line)
 
+class Pytest(Harness):
+    def handle(self, line):
+        if self.handle_script and self.BOOTING_ZHEPHYR in line:
+            with subprocess.Popen([self.handle_script, self.running_dir], stdout=subprocess.PIPE) as proc:
+                try:
+                    stdout, _ = proc.communicate(15)
+                    if "failed" not in stdout.decode():
+                        self.state = "passed"
+                    elif "skip" in stdout.decode():
```

Comment:
Please double confirm if "skip" can be shown with pytest.

Reply:
Never show "skip" with pytest,  but test_ctf.py will show 'skip'  when babeltrace not installed.